setup/setup_broker.sh

```diff
@@ -72,47 +65,44 @@ if [ "${teardown}" != "True" ]; then
     # create dashboard
     gcloud monitoring dashboards create --config-from-file="templates/dashboard.json"
     # create bigquery
-    # bq mk --dataset "${bq_dataset}"
-    # bq mk --table "${bq_dataset}.${alerts_table}" "templates/bq_${survey}_${alerts_table}_schema.json"
-    # bq mk --table "${bq_dataset}.${source_table}" "templates/bq_${survey}_${source_table}_schema.json"
+    alerts_table="alerts"
+    source_table="DIASource"
```

Comment:
These needed to be defined in the section above called "GCP resources used directly in this script"